cmd/repo.go

```diff
@@ -488,13 +493,34 @@ func (index *RepoIndex) buildStacksFromIndex(repoName string, Stacks []Stack) ([
 
 	for id, value := range index.Projects {
 
-		//	Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description, "*" + value[0].DefaultTemplate})
-		Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description})
-
+		Stacks = append(Stacks, Stack{repoName, id, value[0].Version, value[0].Description, "*" + value[0].DefaultTemplate})
 	}
 	for _, value := range index.Stacks {
-		Stacks = append(Stacks, Stack{repoName, value.ID, value.Version, value.Description})
+		Templates := value.Templates
+
+		templatesString := ""
+		if value.Templates != nil {
+			sort.Slice(Templates, func(i, j int) bool { return Templates[i].ID < Templates[j].ID })
+
+			for _, template := range Templates {
+				templateString := ""
+				if template.ID == value.DefaultTemplate {
```

Comment:
this could be simplified a bit by saying tempaltesString += ", " defaultMarker + template.ID.. where default marker is "" or "*" but what u have works too..

Reply:
good idea, changed.